intro-Linux/ch_beyond_the_basics.tex

```diff
@@ -477,11 +477,11 @@ \subsection{Example job script}
  #PBS -q long
  #PBS -m abe
  #PBS -j oe
- export HOMEDIR=$VSC_SCRATCH_VO_USER
+ export HOMEDIR=$VSC_HOME
```

Comment:
@redfast00 Don't teach people to use `$VSC_HOME` as a place to run jobs in, not a good idea.

Please use `$VSC_SCRATCH` instead (it's the `VO` part that is not site-agnostic, VOs are only there in Ghent).